### PR TITLE
Create session cart

### PR DESCRIPTION
**Pull request by bryceholcomb**

### CODE REVIEW

**Comment by bryceholcomb** [comment]
Lets go ahead and merge with the cart model tests passing so we can split up the authorization implementation with the necessary code

File: spec/models/cart_spec.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+require "rails_helper"
+
+RSpec.describe Cart, type: :model do
+  it "has data" do
+    cart = Cart.new({"1" => 3})
+    expect(cart.data).to eq({"1" => 3})
```
Comment (kpearson) [inline spec/models/cart_spec.rb]: _I think those curly braces are redundant._

**Comment by bryceholcomb** [comment]
Cleaned up hound stuff

File: spec/controllers/carts_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+require "rails_helper"
+
+RSpec.describe CartsController, type: :controller do
+  xit "has a session cart which is a hash" do
```
Comment (kpearson) [inline spec/controllers/carts_controller_spec.rb:4]: I think the was this phrased this test belongs in a _sessions controller spec_ or _application controller spec_ as it is called by the before filter.

**Pull request merged**